### ISSUE TITLE
Missing TinyMCE video icon

### ISSUE THREAD

**Issue by dallasbpeters** [keystonejs/keystone#1518]
Hey just noticed this. Thought I would put it on your radar. Missing video icon in the toolbar.

<img width="122" alt="screenshot 2015-07-08 22 38 34" src="https://cloud.githubusercontent.com/assets/91672/8587492/45a4ee94-25c2-11e5-8765-92c3d2d37c65.png">

**Comment by jeffreypriebe** [comment]
To reproduce, add 'media' to both 'wysiwyg additional plugins' and 'wysiwyg additional buttons' in keystone.js of your launching website. Then the TinyMCE will insert the media button.
Sadly, the available wysiwyg icons doesn't appear to include an icon suitable for media / video. See here:
![Available wysiwyg icons](http://i.imgur.com/IbaEVPt.png?3) 

With such an icon, you could do something like what an [intrepid programmer did in this PR](https://github.com/keystonejs/keystone/commit/73ab5db54dae805f0a2a8a5f59e8769c27501299) simply adding the unicode character for the media / video icon to selector for class mce-i-media.

**Workaround** @dallasbpeters Looks like you are already using the image button - otherwise you could reuse that icon. Alternatively, in that PR you'll see where you'd edit the LESS/CSS. If you a rule for mce-i-media then you can set it to any image that you have handy that works for you.
(n.b. LESS recompiles on the fly, the .min.css in the folder can be safely ignored while debugging. [Once in production, I don't know if that's true.].)

Usually it is some sort of ugly film strip icon, [as seen in the red box on the left in this image](https://www.ostraining.com/cdn/images/stories/tutuploadsmedia_1338353173110.png).

To be clear, **it appears that we're short a useful icon for the media/video button** that we could then add to the [Keystone theme for TinyMCE](https://github.com/keystonejs/keystone/blob/master/admin/public/js/lib/tinymce/skins/keystone/less/Icons.less).

**Comment by jeffreypriebe** [comment]
Actually, I spoke a bit hastily. When I went to look for the original source of the fonts to determine where we could get more...
I discovered in admin\public\js\lib\tinymce\skins\keystone\fonts has and "icomoon" font which has a few more icons.
![icomoon-characters](http://i.imgur.com/BmrHVXd.png?1)

Including one for the video/media.
Doing something [like this](https://gist.github.com/jeffreypriebe/fb3f8c869d41eeae6dc4) to Icons.less will yield:
![tinymce-with-media-icon](http://i.imgur.com/BIL2NEe.png?1)

Note: that font-face is from icons.ie7.less - it should really be extracted out and shared into both icon files if you are going to use this font.

**Comment by dallasbpeters** [comment]
Thanks for the help. Not sure I wanna hack keystone internals right now. I'll keep an eye out for a fix or easier way to config the wysiwyg. 

(Thanks for making this awesome tool by they way!)

**Comment by jeffreypriebe** [comment]
I'll second that: thanks to Jed et al for all the work to get keystone to where it is today.

**Comment by hatzipanis** [comment]
Underline is currently also missing.

<img width="71" alt="screen shot 2015-08-21 at 3 29 13 pm" src="https://cloud.githubusercontent.com/assets/1661180/9400762/6adba918-4819-11e5-9a42-bf8d5e544911.png">

**Comment by cameronjroe** [comment]
Was looking into this. How is that .less even compiling? Not seeing it in the gulpfile or server code..?

**Comment by JedWatson** [comment]
@cameronjroe the less is being compiled by `less-middleware`, invoked [here](https://github.com/keystonejs/keystone/blob/master/admin/app/static.js#L45).

Would love a PR fixing this :)

**Comment by cameronjroe** [comment]
Cool, but this less is inside `/public/js/lib/tinymce/skins/keystone/less` which doesn't look like it's required by anything in the `/styles` dir

**Comment by jeffreypriebe** [comment]
@cameronjroe Thanks for the catch on underline, found that when setting up a live site. PR branch updated (along with a few other missing icons).

On the LESS generation, I agree - I haven't been sure if that a "miss" that needs fixing. In admin/app/static.js added in:
```
router.use('/js/lib/tinymce/skins', less(__dirname + '/../public/js/lib/tinymce/skins', lessOptions));
router.use('/js/lib/tinymce/skins/', express.static(__dirname + '/../public/js/lib/tinymce/skins'));
```

(current linting errors on PR are from master)

**Comment by JedWatson** [comment]
Should be closed by #1573, let me know if there's anything else.
